### PR TITLE
Fix typo in ballista bolt description

### PR DESCRIPTION
**Pull request by solarfl4re**
#### Summary

None

### CODE REVIEW

**Comment by NetSysFire** [comment]
Even for changes as trivial as these you must use 

#### Summary

None

**Review by NetSysFire — APPROVED** [review]

**Comment by solarfl4re** [comment]
> Even for changes as trivial as these you must use
> #### Summary
> 
> None

Ok, I've edited the pull request description.

File: data/json/items/ranged/ballista.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "id": "bolt_ballista_steel",
     "type": "AMMO",
     "name": { "str": "ballista bolt" },
-    "description": "A big, steel tipped projectile, twice the size of a normal bolt and much heavier than one.  It gets thicker towards the base, and is fletched there with the portrusions of the shaft itself.",
+    "description": "A big, steel tipped projectile, twice the size of a normal bolt and much heavier than one.  It gets thicker towards the base, and is fletched there with the protrusions of the shaft itself.",
```
Comment (Maleclypse) [inline data/json/items/ranged/ballista.json:37]:
```suggestion
    "description": "A big, steel tipped projectile, twice the size of a normal bolt and much heavier than one.  It gets thicker towards the base, and is fletched there with the protrusions of the shaft itself."
```

File: data/json/items/ranged/ballista.json [hunk 2]
```diff
@@ -34,7 +34,7 @@
     "id": "bolt_ballista_steel",
     "type": "AMMO",
     "name": { "str": "ballista bolt" },
-    "description": "A big, steel tipped projectile, twice the size of a normal bolt and much heavier than one.  It gets thicker towards the base, and is fletched there with the portrusions of the shaft itself.",
+    "description": "A big, steel tipped projectile, twice the size of a normal bolt and much heavier than one.  It gets thicker towards the base, and is fletched there with the protrusions of the shaft itself."
```
Comment (Maleclypse) [inline data/json/items/ranged/ballista.json:37]:
```suggestion
    "description": "A big, steel tipped projectile, twice the size of a normal bolt and much heavier than one.  It gets thicker towards the base, and is fletched there with the protrusions of the shaft itself.",
```

File: data/json/items/ranged/ballista.json (hunk 1 above)
Comment (kevingranade) [inline data/json/items/ranged/ballista.json:37]:
```suggestion
    "description": "A big, steel tipped projectile, twice the size of a normal bolt and much heavier than one.  It gets thicker towards the base, and is fletched there with the protrusions of the shaft itself.",
```

Comment (kevingranade) [inline data/json/items/ranged/ballista.json:37]:
```suggestion
    "description": "A big, steel tipped projectile, twice the size of a normal bolt and much heavier than one.  It gets thicker towards the base, and is fletched there with the protrusions of the shaft itself. ",
```

File: data/json/items/ranged/ballista.json [hunk 3]
```diff
@@ -34,7 +34,7 @@
     "id": "bolt_ballista_steel",
     "type": "AMMO",
     "name": { "str": "ballista bolt" },
-    "description": "A big, steel tipped projectile, twice the size of a normal bolt and much heavier than one.  It gets thicker towards the base, and is fletched there with the portrusions of the shaft itself.",
+    "description": "A big, steel tipped projectile, twice the size of a normal bolt and much heavier than one.  It gets thicker towards the base, and is fletched there with the protrusions of the shaft itself. ",
```
Comment (kevingranade) [inline data/json/items/ranged/ballista.json:37]:
```suggestion
    "description": "A big, steel tipped projectile, twice the size of a normal bolt and much heavier than one.  It gets thicker towards the base, and is fletched there with the protrusions of the shaft itself.",
```

**Pull request merged**